core/jazz_assets/config/global-config.json

```diff
@@ -35,7 +35,25 @@
 		"iam_role",
 		"sqs",
 		"kinesis_stream",
-		"apigee_proxy"
+		"apigee_proxy",
+    "cdn_endpoint_url",
```

Comment:
Fix formatting.

Reply:
now I know why I fixed this twice still like this.  This file use a 2 space tab + 1 space.